### PR TITLE
Add collate template

### PR DESCRIPTION
**Pull request by rolfedh**
Fixes https://github.com/redhat-documentation/vale-at-red-hat/issues/517

### CODE REVIEW

**Comment by rolfedh** [comment]
@aireilly  What would be the best location for this file to get distributed when users run `vale sync`?

File: modules/user-guide/pages/collating-vale-output.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+:description: Collating the Vale CLI report results
+:page-aliases: end-user-guide:collating-vale-output.adoc
+// End of metadata for Antor
+
+:context: collating-vale-output
+:_module-type: CONCEPT
+[id="collating-vale-output_{context}"]
+= Collating Vale output
+
+As a technical writer, you can use a template to collate the results from running `vale` on the command line. This collated output is non-repetitive and might be easier to use.
```
Comment (aireilly) [inline modules/user-guide/pages/collating-vale-output.adoc:13]: Is this "As a technical writer..." construction useful? Anyone can use the Vale at Red Hat style. 

```suggestion
You can use a template to collate the results from running `vale` on the command line. This collated output is non-repetitive and might be easier to use.
```

**Review by aireilly — COMMENTED** [review]
Looks good and useful!

**Comment by aireilly** [comment]
> @aireilly What would be the best location for this file to get distributed when users run `vale sync`?

So, to include this in the vale sync package, you would need to zip this alongside the yml in `.vale/styles/RedHat`. I'm not sure this is the right way to handle this though. Do we really want to include an optional output template alongside rules? We could provide a script to pull down the template maybe.

**Comment by aireilly** [comment]
> > @aireilly What would be the best location for this file to get distributed when users run `vale sync`?
> 
> So, to include this in the vale sync package, you would need to zip this alongside the yml in `.vale/styles/RedHat`. I'm not sure this is the right way to handle this though. Do we really want to include an optional output template alongside rules? We could provide a script to pull down the template maybe.

On second thoughts, we already include an adoc file in the RH rules. I am OK to include this template there :)

File: modules/user-guide/pages/collating-vale-output.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+The `RedHat` style includes a `collate-output.tmpl` template you can specify when you run the `vale` command:
+
+----
+$ cd openshift-docs/logging/troubleshooting/
+$ vale --output='~/styles/collate-output.tmpl' cluster-logging-troubleshooting-for-critical-alerts.adoc
```
Comment (aireilly) [inline modules/user-guide/pages/collating-vale-output.adoc:19]:
```suggestion
$ vale --output='./.vale/styles/RedHat/collate-output.tmpl' logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc
```

File: modules/user-guide/pages/collating-vale-output.adoc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+:_module-type: CONCEPT
+[id="collating-vale-output_{context}"]
+= Collating Vale output
+
+As a technical writer, you can use a template to collate the results from running `vale` on the command line. This collated output is non-repetitive and might be easier to use.
+
+The `RedHat` style includes a `collate-output.tmpl` template you can specify when you run the `vale` command:
+
+----
+$ cd openshift-docs/logging/troubleshooting/
```
Comment (aireilly) [inline modules/user-guide/pages/collating-vale-output.adoc:18]:
```suggestion
$ cd openshift-docs/
```

File: modules/user-guide/pages/collating-vale-output.adoc (hunk 2 above)
Comment (aireilly) [reply modules/user-guide/pages/collating-vale-output.adoc:19]: The idea here is that we `vale sync` in `~/openshift-docs`, and then run Vale from the root project dir, this allows us to use the template using a relative path.

Comment (aireilly) [reply modules/user-guide/pages/collating-vale-output.adoc:19]: depending on how you config Vale I guess

File: modules/user-guide/pages/collating-vale-output.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/user-guide/pages/collating-vale-output.adoc:13]: Updated. Thanks.

File: modules/user-guide/pages/collating-vale-output.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/user-guide/pages/collating-vale-output.adoc:19]: Updated. Thanks.

File: modules/user-guide/pages/collating-vale-output.adoc (hunk 3 above)
Comment (rolfedh) [reply modules/user-guide/pages/collating-vale-output.adoc:18]: Updated. Thanks.

File: modules/user-guide/pages/collating-vale-output.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/user-guide/pages/collating-vale-output.adoc:19]: I'm not sure this works. Let's talk about it next week.

**Comment by rolfedh** [comment]
@aireilly I've addressed the issues we discussed and verified the content. I think this is ready to merge.

**Pull request merged**